pom.xml

```diff
@@ -17,41 +17,180 @@
+                <executions>
+                    <execution>
+                        <id>compile</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>compile</goal>
+                        </goals>
+                        <configuration>
+                            <sourceDirs>
+                                <source>src/main/kotlin</source>
+                                <source>src/main/java</source>
+                            </sourceDirs>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>test-compile</id>
+                        <phase>test-compile</phase>
+                        <goals>
+                            <goal>test-compile</goal>
+                        </goals>
+                        <configuration>
+                            <sourceDirs>
+                                <source>src/test/kotlin</source>
+                                <source>src/test/java</source>
+                            </sourceDirs>
+                        </configuration>
+                    </execution>
+                </executions>
+                <configuration>
+                    <jvmTarget>1.8</jvmTarget>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>4.2.1</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <_removeheaders>Bnd-*, Tool, Require-Capability, Include-Resource
+                        </_removeheaders>
+                        <!--
+                        Prevent importing a specific version. Some libraries
+                        like Guava rapidly change major versions, which is a pain for
+                        downstream consumers wanting a different/newer version than
+                        used here.
+                        -->
+                        <Import-Package>*;version=!</Import-Package>
+                        <!-- This is necessary for free/extensions functions defined in the kotlin code to be
+                        visible to kotlin code using this lib.  See https://youtrack.jetbrains.com/issue/KT-9164. -->
+                        <Include-Resource>
+                            /=src/main/resources/,
+                            META-INF=target/classes/META-INF
+                        </Include-Resource>
+                    </instructions>
+                </configuration>
+            </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.8.1</version>
+                <version>3.5.1</version>
```

Comment:
Change from 3.8.1 to 3.5.1?

Reply:
Nice catch, must have been this version in the submodule file. Fixed.